lib/internal/modules/cjs/loader.js

```diff
@@ -231,6 +232,10 @@ ObjectDefineProperty(Module, 'wrapper', {
   }
 });
 
+ObjectDefineProperty(Module.prototype, 'isPreloading', {
```

Comment:
Is it intentionally non configurable?

Reply:
Eh, only in that I didn't see an explicit need to make it configurable. That is, Node.js owns this state and user code really shouldn't be trying to mutate it that way.

Reply:
ok, I just remember that I had some issue in the past with adding a non configurable property on EventEmitter even it's formal correct as you say.